spec/fake_braintree/customer_spec.rb

```diff
@@ -114,6 +114,18 @@
     expect(billing_address.street_address).to eq '1 E Main St'
     expect(billing_address.postal_code).to eq '60622'
   end
+
+  it "sets the creation time" do
+    customer = Braintree::Customer.create(
+      credit_card: {
+        number: TEST_CC_NUMBER,
+        expiration_date: '04/2016'
+      }
+    ).customer
+
+    creation_time = Time.parse(customer.created_at)
+    expect(creation_time).to be_within(1).of(Time.now)
```

Comment:
Do not use `Time.now` without zone. Use one of `Time.zone.now`, `Time.now.current`, `Time.now.in_time_zone`, `Time.now.utc`, `Time.now.getlocal`, `Time.now.iso8601`, `Time.now.jisx0301`, `Time.now.rfc3339`, `Time.now.to_i`, `Time.now.to_f` instead.